package.json

```diff
@@ -30,6 +30,7 @@
   },
   "devDependencies": {
     "enzyme": "^2.8.2",
+    "lodash": "^4.17.4",
```

Comment:
Let's go ahead and add this as a main dependency because it's used in a component